app/src/androidTest/java/com/mapbox/maps/testapp/annotation/UpdateAnnotationTest.kt

```diff
@@ -78,11 +78,15 @@ class UpdateAnnotationTest : BaseMapTest(), OnMapLoadedListener {
               .withTextTransform(TextTransform.UPPERCASE)
               .withTextSize(15.0)
               .withTextRadialOffset(1.0)
-              .withTextOpacity(0.8)
               .withTextOffset(listOf(1.0, 2.0))
               .withTextMaxWidth(10.0)
               .withPoint(Point.fromLngLat(0.0, 0.0))
           )
+          for (i in 0..100) {
```

Comment:
Does it make sense to move it to separate test?

Reply:
I would prefer to add it here as this case is used to verify annotation update and in this loop, we are updating this annotation.
Another reason is that with fewer test cases we could have a faster test run 😄